packages/framework/pr_tools/LaunchDriver.py

```diff
@@ -118,6 +118,8 @@ def main(argv):
 
   if args.extra_configure_args:
      cmd += f" --extra-configure-args=\"{args.extra_configure_args}\""
+  elif os.getenv("EXTRA_CONFIGURE_ARGS"):
+     cmd += f" --extra-configure-args=\"{os.getenv('EXTRA_CONFIGURE_ARGS')}\""
```

Comment:
I'm picking a nit here, but do you want it to be one or the other?  What if we set the environment variable _and_ pass the argument?  Will we remember that the argument overrides the environment variable?  I think it would be better to have the argument add on to the environment value, or to at least print that the environment value is being ignored in favor of the command line.  Do you think it's worth it, or am I overly concerned?